recipe/meta.yaml

```diff
@@ -66,6 +66,7 @@ requirements:
     - {{ compiler('cxx') }}
     - {{ mpi }}  # [mpi == 'openmpi' and build_platform != target_platform]
   host:
+    - m4  # dev only
```

Comment:
I would move `m4` from `host` to `build`.  That's usually where it is in other packages and it fits with `make`, `cmake`, etc.  That might account for the platform issue you're seeing.

Reply:
Indeed. Thanks!